github.go

```diff
@@ -10,6 +10,8 @@ import (
 	"github.com/gin-gonic/gin"
 	"github.com/google/go-github/github"
 	"github.com/iver-wharf/wharf-api-client-go/pkg/wharfapi"
+	"github.com/iver-wharf/wharf-core/pkg/ginutil"
+	_ "github.com/iver-wharf/wharf-core/pkg/problem"
```

Comment:
Why was this import for `github.com/iver-wharf/wharf-core/pkg/problem` added as a discard import?

Reply:
It seemed like it was necessary to have it imported so that `problem.Response` would be recognized as a type in e.g.
```
// @Failure 400 {object} problem.Response "Bad request"
// @Failure 401 {object} problem.Response "Unauthorized or missing jwt token"
// @Failure 502 {object} problem.Response "Bad gateway"
```

Reply:
Now that I'm using VS Code it doesn't seem to be an issue.

Reply:
If you remove it, can you still do `swag init`? Maybe it was an issue with when the wharf-core module was not referenced at all but since you're referencing the `ginutil` that in turn references the `problem` package maybe that's enough?

Reply:
Right!
Yeah it wouldn't work when I did swag init before, same thing now. It complains about not finding the type definition for problem.Response.

It does work with `--parseDepth=2` though, in which case I'm assuming it finds it through the `ginutil` package.

Reply:
Then I suggest you update the README and rely on it finding it via the `ginutil` package.